src/domain/collection/curatedEventList/__tests__/CuratedEventList.test.tsx

```diff
@@ -21,19 +23,62 @@ const collection = fakeCollection({
   curatedEvents,
 }) as CollectionFieldsFragment;
 
-const getMocks = (events: EventFieldsFragment[]) => [
+const getMocks = (events: EventFieldsFragment[], ids = eventIds) => [
   {
     request: {
       query: EventsByIdsDocument,
       variables: {
-        ids: eventIds,
+        ids,
         include: ['location'],
       },
     },
     result: { data: { eventsByIds: events } },
   },
 ];
 
+// Creates array of mocks to match pagination queries
+const getMocksForPagination = (eventsCount = 35) => {
+  const eventIds = range(eventsCount).map((i) => (i + 1).toString());
+  const eventNames = eventIds.map((id) => `Event ${id}`);
+  const curatedEvents = eventIds.map(
+    (id) => `http://localhost:3000/fi/event/${id}`
+  );
```

Comment:
looks like it can be only one map instead of three, but might be not that readable then.

Reply:
eventIds for example are needed later in the code. Maybe the could be array that that has objects that contains eventId, eventName and curatedEvents

Reply:
but then again I don't know if it makes it less readable

Reply:
refactored the function little bit by using flow